fic/provider_test.go

```diff
@@ -91,6 +93,17 @@ func testAccPreCheckRouterToECLConnection(t *testing.T) {
 	}
 }
 
+func testAccPreCheckAzureConnection(t *testing.T) {
+	testAccPreCheckRequiredEnvVars(t)
+
+	if OS_AZURE_SERVICE_KEY == "" {
```

Comment:
Please make either one of following change.

- If you want to skip when env var is not set, please change message like "This environment only runs ... tests"
- If you want to make tests fail, change method from Skip to Fatal

Reply:
Fixed!